package.json

```diff
@@ -33,7 +33,8 @@
   },
   "homepage": "https://github.com/buttercup/credentials#readme",
   "dependencies": {
-    "@buttercup/signing": "^0.1.0"
+    "@buttercup/signing": "^0.1.0",
+    "hash.js": "1.1.7"
```

Comment:
This version shouldn't be hardcoded.. it's fine to use the `^` prefix :)

Reply:
hah I always forget to change, when I use npm install